sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImpl.java

```diff
@@ -755,7 +755,12 @@ public void deleteDataset(String projectId, String datasetId)
           out.setJson(row.getUnknownKeys());
           rows.add(out);
 
-          dataSize += row.toString().length();
```

Comment:
This was incorrect. The length `toString()` was never the same as the length of JSON.